src/server/pfs/server/driver.go

```diff
@@ -3578,6 +3603,20 @@ func (d *driver) getFile(pachClient *client.APIClient, file *pfs.File, offset in
 		}
 
 		// retrieve the content of all objects in 'objects'
+		if len(brs) > 0 {
```

Comment:
If I am understanding this correctly, a user would not be able to append new data to a file copied from an output repo? Is this acceptable behavior? I think put file would still store the data, but only the copied data would be accessible.

Reply:
Yeah, it's a bit suboptimal, but I think it's an ok compromise to make, because otherwise we'll have to extend the data structure to capture the ordering between the block refs and the objects that are referenced. And I think it's a pretty uncommon use case to try to copy a file from the output repo, and then want to append to it later.

Reply:
Yeah, I think the options are some support for copying from output to input repos with some strange semantics or no support. If you think the former makes the most sense, then I think this is fine, but it could be useful to make copied files from output to input repos immutable (just return an error for put files that have been copied from output to input repos).

Reply:
I think an error here makes sense. They wouldn't be immutable though, since you can still append other output files to them.

Reply:
Error added.